src/Components/ChatContainer/ChatContainer.js

```diff
@@ -9,11 +9,10 @@ import { Redirect } from 'react-router';
 
 const endPoint = "http://localhost:5000";  
 
-function ChatContainer({ groupCode, roomCode }) {
+function ChatContainer({ groupCode, roomCode, name }) {
   const socket = io.connect(`${endPoint}`);
   const [messages, setMessages] = useState([]);
-  const [groupDetails, setGroupDetails] = useState({});
-  const [room, setRoom] = useState('');
+  const [roomData, setRoomData] = useState('');
```

Comment:
room and group details are contained w/in roomData?

Reply:
Room data was replaced with room details, and groupDetails isn't apart of the MVP for right now